src/v0/sources/shopify/transform.js

```diff
@@ -198,15 +205,31 @@ const processEvent = async (inputEvent, metricMetadata) => {
   return message;
 };
 
-const isIdentifierEvent = (event) => event?.event === 'rudderIdentifier';
-
+const isIdentifierEvent = (event) => ['rudderIdentifier', 'rudderSessionIdentifier'].includes(event?.event);
 const processIdentifierEvent = async (event, metricMetadata) => {
   if (useRedisDatabase) {
-    const lineItemshash = getHashLineItems(event.cart);
-    const value = ['anonymousId', event.anonymousId, 'itemsHash', lineItemshash];
+    let value;
+    if (event.event === 'rudderIdentifier') {
+      const lineItemshash = getHashLineItems(event.cart);
+      value = ['anonymousId', event.anonymousId, 'itemsHash', lineItemshash];
+      /*
+      cart_token: {
+      anonymousId:"anon_id1",
+      lineItemshash:"0943gh34pg"
+     }
+      */
+    } else {
+      value = ['sessionId', event.sessionId];
+      /*
+      cart_token: {
+      sessionId: "session_id1"
+     }
+     */
+    }
     try {
       await RedisDB.setVal(`${event.cartToken}`, value);
     } catch (e) {
+      logger.debug(`{{SHOPIFY::}} cartToken maping set call Failed due redis error ${e}`);
```

Comment:
`logger.debug(`{{SHOPIFY::}} Failed to set cartToken mapping due to a Redis error: ${e}`);`